recipes/open62541/all/test_package/test_package.c

```diff
@@ -1,75 +1,11 @@
 #ifdef UA_ENABLE_AMALGAMATION
 #include <open62541.h>
 #else
-#include <open62541/plugin/log_stdout.h>
 #include <open62541/server.h>
 #include <open62541/server_config_default.h>
-#include <open62541/network_tcp.h>
 #endif
-
-/* Files namespace_foo_flt_generated.h and namespace_foo_flt_generated.c are created from FooFlt.NodeSet2.xml in the
- * /src_generated directory by CMake */
-#include "foo_flt_nodeids.h"
-#include "namespace_foo_flt_generated.h"
-#include <open62541/plugin/network.h>
-#include <signal.h>
 #include <stdlib.h>
 
-UA_Boolean running = false;
-
-static void server_stop_callback(UA_Server *server, void *data) {
-    UA_LOG_INFO(UA_Log_Stdout, UA_LOGCATEGORY_SERVER, "stopping server due to timer");
-    running = false;
-}
-
 int main(void) {
     UA_Server *server = UA_Server_new();
```

Comment:
is there a "delete" function that handles the pointer cleanup? If there is one, I would call it to free the pointer. I think OSs will free the resources upon process termination, but I would still do the minimal thing.